### PR TITLE
custom save file name

### PR DESCRIPTION
**Pull request by disseminate**

### CODE REVIEW

File: NitroxServer/App.config [hunk 1]
```diff
@@ -10,5 +10,6 @@
         <add key="GameMode" value="Survival"/>
         <add key="DisableConsole" value="false"/>
         <add key="ServerAdminPassword" value=""/>
+        <add key="SaveFile" value="save.nitrox"/>
```
Comment (AlexejheroYTB) [inline NitroxServer/App.config]: Maybe make the save name configurable, but enforce the `.nitrox` at the end of the string?

Comment (DerTyan) [reply NitroxServer/App.config]: Maybe change the setting to something like world name and then combine the world name and the file extension as the save file.

Comment (AlexejheroYTB) [reply NitroxServer/App.config]: Yea thats what I meant

Comment (disseminate) [reply NitroxServer/App.config]: Done

**Pull request merged**